src/models/oryp12/repairs.md

```diff
@@ -0,0 +1,250 @@
+# Oryx Pro (Parts & Repairs)
+
+Many components in your Oryx Pro can be upgraded or replaced as necessary.
+
+- [Removing the bottom cover](#removing-the-bottom-cover)
+- [Replacing the RAM](#replacing-the-ram)
+- [Replacing an M.2/NVMe SSD](#replacing-an-m2nvme-ssd)
+- [Replacing the WiFi/Bluetooth module](#replacing-the-wireless-card)
+- [Replacing the battery](#replacing-the-battery)
+- [Replacing the CMOS battery](#replacing-the-cmos-battery)
+- [Replacing the fans/heatsink/thermal paste](#replacing-the-cooling-system)
+- [Replacing the speakers](#replacing-the-speakers)
+- [Replacing the keyboard](#replacing-the-keyboard)
+
+## Removing the bottom cover:
+
+Removing the cover is required to access the internal components. Prior to removing the cover, ensure the AC power is unplugged and all peripherals (including SD cards and USB drives) are unplugged or removed from the system.
+
+**Part numbers:**
+- Bottom panel: `6-39-PE603-014-1`
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 5 minutes  
+**Difficulty:** Easy <span style="color:green;">●</span>
+
+### Steps to remove the bottom cover:
+
+1. Place the machine lid-side down.
+    - Use a soft surface (such as a towel) to avoid scratches.
+2. Remove the 16 bottom panel screws.
+
+![Bottom panel screws](./img/bottom-panel-screws.webp)
+
+3. Pull the bottom panel off, starting from the hinges in the back.
+
+## Replacing the RAM:
+
+The Oryx Pro 12 supports up to 96GB (2x48GB) of DDR5 SO-DIMMs running at 5600MHz. If you've purchased new RAM, need to replace your RAM, or are reseating your RAM, follow these steps.
```

Comment:
This should be 5200Mhz as the firmware does not support anything above that.

Reply:
That is not correct. Oryp11 was limited to 5200MHz because its CPU was a 13900H. Oryp12 uses a 14900HX, which supports 5600MHz.